apps/examples/wc-angular12/angular.json

```diff
@@ -100,5 +100,8 @@
             }
         }
         }
+    },
```

Comment:
This was caused by running yarn dev and opting out of recording analytics - I wouldn't worry about it :)